cypress/pages/Menu.js

```diff
@@ -1,5 +1,10 @@
 export class Menu {
+  /**
+   * @param {('Custom Types'|'Slices'|'Changes')} label - The menu item to click
+   */
   navigateTo(label) {
-    return cy.get("aside").contains(label).click();
+    cy.get("aside").contains(label).click();
+    cy.get("main a").contains(label);
```

Comment:
what would that one check ?

Reply:
To identify the page title in the main panel: 
<img width="265" alt="image" src="https://user-images.githubusercontent.com/6433957/217313745-8183f701-04d2-4842-985d-1a4b7f7d2f54.png">
It would have been nice to have a `h1` or `h2` to locate this but it didn't exist. Maybe we could just add some aria labels like `role="heading" aria-level="1"` as described [here](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/heading_role), wdyt?

Reply:
ah i see.
I thought it would an element of the menu itself that's why I found it confusing.

Reply:
you're right, it shouldn't be part of the Menu page object, I'll move it away.

Reply:
done